### PR TITLE
Add support for stack classifier and regressor

### LINKED ISSUE #468: Add 'sklearn.ensemble._stacking.StackingClassifier'
A user requested us to add `sklearn.ensemble._stacking.StackingClassifier`.  (See [doc](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.StackingClassifier.html)).  

At first glance this appears it should be reasonably easy to implement/similar to our other classifier converters.

### PR DESCRIPTION
**Pull request by interesaaat**
Closes #468

### CODE REVIEW

File: hummingbird/ml/_parse.py [hunk 1]
```diff
@@ -27,6 +27,11 @@
 from .operator_converters import constants
 from .supported import get_sklearn_api_operator_name, get_onnxml_api_operator_name, get_sparkml_api_operator_name
 
+try:
+    from sklearn.ensemble import StackingClassifier, StackingRegressor
+except ImportError:
+    StackingClassifier = None
```
Comment (ksaur) [inline hummingbird/ml/_parse.py:34]: Can you please put some comment like below on "StackingClassifier not available in scikit-learn < 0.22" so we dont' forget later when we hopefully clean this

File: tests/test_sklearn_pipeline.py [hunk 2]
```diff
@@ -471,7 +478,6 @@ def test_pipeline_column_transformer_drop_noweights(self):
             model.predict_proba(X_test), torch_model.predict_proba(X_test.values), rtol=1e-06, atol=1e-06,
         )
 
-    @unittest.skipIf(ColumnTransformer is None, reason="ColumnTransformer not available in 0.19")
```
Comment (ksaur) [inline tests/test_sklearn_pipeline.py:474]: Did you pull this b/c  it's now part of `_parse_sklearn_stacking`?  or do these tests just happen to not hit it?

Comment (interesaaat) [reply tests/test_sklearn_pipeline.py:474]: We only support Sklearn >= 0.21.3 so this is redundant.

File: hummingbird/ml/_parse.py (hunk 1 above)
Comment (interesaaat) [reply hummingbird/ml/_parse.py:34]: yep.

**Pull request merged**